components/Header.js

```diff
@@ -0,0 +1,24 @@
+// components/Header.js
```

Comment:
I think we know which file we're in without this comment 😄

Reply:
Also I think we should stick with `tsx` files unless it's not feasible! This file should just work by changing the extension without needing to change anything else